server/src/routes/clientRouter.js

```diff
@@ -3,7 +3,7 @@ import getClientIndexPath from "../config/getClientIndexPath.js";
 
 const router = new express.Router();
 
-const clientRoutes = ["/", "/user-sessions/new", "/users/new"];
+const clientRoutes = ["/", "/user-sessions/new", "/users/new","/characters"];
```

Comment:
We should add a space between comma separated values

Although you might decide to start listing the routes vertically as your app grows

```js
const clientRoutes = [
  "/", 
  "/user-sessions/new", 
  "/users/new",
  "/characters"
];
```